test/ExecutionFramework.h

```diff
@@ -51,6 +51,25 @@ class ExecutionFramework
 {
 
 public:
+	struct log_record
```

Comment:
This is the same as in MockHost except for the index - do we need the duplication?